CRM/Core/Payment/PayPalImpl.php

```diff
@@ -694,7 +694,7 @@ public function isSupported($method) {
       // by standard or express.
       return FALSE;
     }
-    return $this->_paymentProcessor->supports($method);
+    return parent::isSupported($method);
```

Comment:
@mattwire just pointing out I had to revert this change to make it work (the fact things like this can sneak in is why I try to get each PR down to as limited a change set as possible, otherwise it's too easy to miss something seemingly innocuous)